src/test/java/tests/TurboIssueTests.java

```diff
@@ -111,4 +111,80 @@ public void testCombinePullRequest() {
         assertEquals(issue3.getUpdatedAt(), newIssue3.getUpdatedAt());
         assertEquals(issue4.getUpdatedAt(), newIssue4.getUpdatedAt());
     }
+
+    /**
+     * Tests that isLabelsSame returns false with an empty list argument while the issue has some labels
+     */
+    @Test
+    public void testIsLabelsSameEmptyList() {
+        TurboIssue issue = createIssueWithLabels(Arrays.asList("label1", "label2"));
+        assertFalse(issue.isLabelsSame(new ArrayList<>()));
+    }
+
+    /**
+     * Tests that isLabelsSame returns false when the argument labels don't overlap with the issue's labels
+     */
+    @Test
+    public void testIsLabelsSameNonOverlapping() {
+        TurboIssue issue = createIssueWithLabels(Arrays.asList("label1", "label2"));
+        List<String> labels = Arrays.asList("label3", "label4");
+        assertFalse(issue.isLabelsSame(labels));
+    }
+
+    /**
+     * Tests that isLabelsSame returns true when the argument labels are same as the issue's labels
+     */
+    @Test
+    public void testIsLabelsSameNoChange1() {
+        List<String> labels = Arrays.asList("label1", "label2");
+        TurboIssue issue = createIssueWithLabels(Arrays.asList("label1", "label2"));
+        assertTrue(issue.isLabelsSame(labels));
+    }
+
+    /**
+     * Tests that isLabelsSame returns true when both the argument labels and the issue's labels are empty
+     */
+    @Test
+    public void testIsLabelsSameNoChange2() {
+        List<String> labels = Arrays.asList("label1", "label2");
+        TurboIssue issue = createIssueWithLabels(Arrays.asList("label1", "label2"));
+        assertTrue(issue.isLabelsSame(labels));
+    }
+
+    /**
+     * Tests that isLabelsSame return false when the issue's labels are a strict subset of the argument labels
+     */
+    @Test
+    public void testIsLabelsSameSubset1() {
+        TurboIssue issue = createIssueWithLabels(Arrays.asList("label1", "label2"));
+        List<String> labels = Arrays.asList("label1", "label2", "label3");
+        assertFalse(issue.isLabelsSame(labels));
+    }
+
+    /**
+     * Tests that isLabelsSame return false when the argument labels are a strict subset of the issue's labels
+     */
+    @Test
+    public void testIsLabelsSameSubset2() {
+        TurboIssue issue = createIssueWithLabels(Arrays.asList("label1", "label2", "label3"));
+        List<String> labels = Arrays.asList("label1", "label2");
+        assertFalse(issue.isLabelsSame(labels));
+    }
+
+    /**
+     * Tests that isSameLabels returns false when the argument labels partially overlap with the issue's labels
+     */
+    @Test
+    public void testIsLabelsSamePartialOverlap() {
+        TurboIssue issue = createIssueWithLabels(Arrays.asList("label1", "label2", "label3"));
+        List<String> labels = Arrays.asList("label2", "label3", "label4");
+        assertFalse(issue.isLabelsSame(labels));
+    }
```

Comment:
Not sure if it's worth having many test methods to test this simple method. Can you write a utility method like this?
`verifyIsLabelsSame(List<String> originalLabels, List<String> newLabels, boolean expected)`
Then you can call it multiple times in the same `@Test`.